src/modules/categories/components/CategoryListItem.js

```diff
@@ -41,9 +41,10 @@ class CategoryListItem extends React.Component<PropsType> {
   renderSubCategories (): Array<React.Node> {
     const {subCategories} = this.props
     return subCategories.map(subCategory =>
-      <SubCategoryListItem key={subCategory.id}
-                           subCategory={subCategory} onItemPress={this.props.onItemPress}
-                           theme={this.props.theme} />
+      <View key={subCategory.path}>
```

Comment:
Why do we need the `View` as a wrapper here?

Reply:
Done